cost-analyzer/values.yaml

```diff
@@ -857,6 +857,10 @@ kubecostDeployment:
     storageClass: ""
     databaseVolumeSize: 100Gi
     configVolumeSize: 1Gi
+    busyBoxImage:
```

Comment:
@srpomeroy Now that we've updated `query-service-deployment-template.yaml` we should remove these lines from `values.yaml`?

Reply:
Good catch!